src/Inventory/InventoryList.js

```diff
@@ -192,4 +219,23 @@ export const styles = StyleSheet.create({
 	rightButton: {
 		padding: 5
 	},
+	standaloneRowFront: {
```

Comment:
Should we just add this to the `config/styles` directory so it can be reused in other components? e.g. Grocery stuff

Reply:
Yeah I can do that